File: SecondYearCS/OOP/CivitasRB/lib/CasillaDescanso.rb
require_relative 'sorpresa.rb'
require_relative 'casilla.rb'
require_relative 'tipo_sorpresa.rb'
require_relative 'titulo_propiedad.rb'
require_relative 'mazo_sorpresa.rb'
require_relative 'diario.rb'
require_relative 'dado.rb'
module CivitasRB
  class CasillaDescanso < Casilla
    def initialize(nombre)
      super(nombre)
    end

    def recibeJugador(act, todos)
      self.informe(act, todos)
    end

  end
end

File: SecondYearCS/OOP/Civitas/src/civitas/CasillaDescanso.java
package civitas;
import java.util.ArrayList;

public class CasillaDescanso extends Casilla {

    CasillaDescanso ( String nombre ) {
        super(nombre);
    }
    
    @Override
    void recibeJugador (int actual, ArrayList<Jugador> todos){
        informe(actual,todos);
    };

    @Override
    public String toString(){
       String result = super.toString();
       return result;
    }

}
